guides/md/keras_cv/cut_mix_mix_up_and_rand_augment.md

```diff
@@ -421,8 +415,15 @@ with strategy.scope():
 
 <div class="k-default-codeblock">
 ```
-32/32 [==============================] - ETA: 0s - loss: 4.8694 - accuracy: 0.0049
-32/32 [==============================] - 399s 12s/step - loss: 4.8694 - accuracy: 0.0049 - val_loss: 43.5487 - val_accuracy: 0.0187
+WARNING:tensorflow:There are non-GPU devices in `tf.distribute.Strategy`, not using nccl allreduce.
```

Comment:
Let's remove these warnings.

Reply:
Sounds good.  unfortunately I don’t think tf.distribute has an API to mute these

Reply:
That's too bad, but manual edits sound good.